starter-code/models/User.js

```diff
@@ -0,0 +1,25 @@
+const mongoose = require("mongoose");
+
+const userSchema = new mongoose.Schema(
+  {
+    username: {
+      type: String,
+      required: "Username is required",
+      unique: true,
+      index: true
+    },
+    password: {
+      type: String,
+      required: "Password is required"
+    }
+  },
+  {
+    timestamps: true
+  }
+);
+
+const model = mongoose.model("User", userSchema);
+
+model.collection.createIndexes();
```

Comment:
El error que os sale por consola tiene que ver con la llamada a `createIndexes`. Lo resolvió Marc en clase el otro día, si tenéis dudas preguntadnos a los TA's.